clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/ops/servergroup/DeployOpenstackAtomicOperation.groovy

```diff
@@ -177,4 +184,20 @@ class DeployOpenstackAtomicOperation implements AtomicOperation<DeploymentResult
     result
   }
 
+  /**
+   * Return the file contents of a template, either from the account config location or from the classpath.
+   * @param filename
+   * @return
+   */
```

Comment:
It feels like this should be cached ... I don't think we should be reading the file in on each server group creation.